File: c/tmp/random/FizzBuzz.c
#include <stdio.h>

/**
 * main - Fizz Buzz
 *
 * Return: 0 (success)
 */
int main(void)
{
	int i;

	for (i = 0; i < 20; i++)
	{
		if (i == 0)
			printf("%d ", i);
		else if (i % 15 == 0)
			printf("FizzBuzz ");
		else if (i % 3 == 0)
			printf("Fizz ");
		else if (i % 5 == 0)
			printf("Buzz ");
		else
			printf("%d ", i);
	}
	putchar ('\n');

	return (0);
}

File: dsa/fizzbuzz.py
#!/usr/bin/env python3
"""A fizzbuzz algorithm implementation"""
import sys

def fizzbuzz(count):
    """fizzbuzz implementantion"""
    if count <= 0:
        return

    for num in range(1, count + 1):
        if num % 15 == 0:
            print("FizzBuzz")
        elif num % 3 == 0:
            print("Fizz")
        elif num % 5 == 0:
            print("Buzz")
        else:
            print(num)


if __name__ == "__main__":
    if len(sys.argv) == 2:
        try:
            count = int(sys.argv[1])
        except Exception as e:
            print("Count must be a number")
            exit(1)
    else:
        print(f"Usage: ./fizzbuzz.py count")
        exit(1)

    fizzbuzz(count)
